tests/suites/model/migration.sh

```diff
@@ -66,6 +66,16 @@ run_model_migration_version() {
 	export JUJU_VERSION=$juju_version_without_build_number
 	major_minor=$(echo "$JUJU_VERSION" | cut -d'-' -f1 | cut -d'.' -f1,2)
 
+	# Google cloud is slow sometimes to operate with charmhub. So, we need to enlarge the timeout for wait_for.
+	if [[ "${BOOTSTRAP_PROVIDER:-}" == "gce" ]]; then
+    wait_for_timeout=1800
+    wait_for_timeout_etcd_app=1800
+  else
+    wait_for_timeout=600
+    wait_for_timeout_etcd_app=900
+  fi
```

Comment:
Lets just set the timeout to 1800 for all providers. If we still encounter this issue, it is likely a cloud or a charm issue.

Reply:
ok, agreed.